LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2015-2016 Vincent Hanquez <vincent@snarc.org>
+Copyright (c) 2015-2016 Vincent Hanquez <vincent@snarc.org> 2016-2017
```

Comment:
that's a weird format, usually the year is before the name and after the diff it's on both side. It would probably make sense to just drop the years, that hasn't been required for many many years now ..